experimental/packages/exporter-trace-otlp-http/test/browser/CollectorTraceExporter.test.ts

```diff
@@ -191,7 +197,7 @@ describe('OTLPTraceExporter - web', () => {
         collectorTraceExporter.export(spans, () => {
         });
 
-        setTimeout(() => {
+        nextTick(() => {
```

Comment:
Ditto. Probably we can use `queueMicrotask` here.

Reply:
setTimeout with a time of 0 would probably also work